drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -380,6 +397,9 @@ static int sdw_compute_bus_params(struct sdw_bus *bus)
 		return -EINVAL;
 	}
 
+	mstr_prop->default_col = curr_dr_freq / mstr_prop->default_frame_rate /
+					mstr_prop->default_row;
```

Comment:
nitpick: I would align this with the curr_dr_freq.

Reply:
done